plugin.yaml

```diff
@@ -10,58 +10,215 @@ plugins:
     # URL to archive containing the plugin or name of directory containing
     # the plugin if it is included in the the blueprint directory under the
     # "plugins" directory.
-    source: https://github.com/cloudify-cosmo/cloudify-gcp-plugin/archive/master.zip
+    source: https://github.com/szpotona/cloudify-gcp-plugin/archive/add_keypair.zip
 
 node_types:
   cloudify.gcp.nodes.Instance:
     derived_from: cloudify.nodes.Compute
+    properties:
+      use_external_resource:
```

Comment:
I think the resource_id is missing here, but use_external_resource requires it... The same is also true for all other occurences of use_external_resource in this yaml